packages/contentful--create-contentful-app/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
```

Comment:
Can we just use `@tsconfig/node16` here instead. Thay way we are sure that we always have the correct `target` and `lib` set up: https://www.npmjs.com/package/@tsconfig/node16